config/config/common-config.yaml

```diff
@@ -170,7 +170,6 @@ prometheus:
 thanos:
   # Enables Thanos components.
   # If this isn't set, no components will be deployed
-  enabled: false
```

Comment:
I think we should still be able to disable thanos. As we can do for most of the services we deploy. Specially then deploying a small dev cluster for testing something which is not dependent on thanos.

Reply:
Í'm a bit split on if we should be able to disable thanos or not. Without it we cannot get any wc metrics to grafana. But I agree that there might be some cases where that is ok and it would be worth dropping that extra load.

Reply:
I agree that for any real environment that thanos should not be disabled. This thought was just when setting up a dev environment. Also I prefer to have the possibility to disable services and features if they are misbehaving during development.

Reply:
I added back the way to configure enabling thanos